xla/stream_executor/cuda/cuda_dnn.h

```diff
@@ -735,8 +736,10 @@ absl::StatusOr<CudnnGraph> GetCudnnFlashAttentionBackwardOperationGraph(
     const std::optional<dnn::TensorDescriptor> bias_descriptor,
     const std::optional<dnn::TensorDescriptor> dbias_descriptor,
     std::optional<double> dropout_rate, std::optional<int64_t> seed,
-    double scale, bool use_dropout, bool use_bias, dnn::FMHAMaskKind mask_type,
-    bool force_deterministic, int sliding_window_length, int max_seg_per_batch);
+    double scale, bool use_dropout, bool use_bias,
+    const dnn::FMHAMaskKind mask_type, bool force_deterministic,
+    const int sliding_window_length, const int max_seg_per_batch,
+    std::shared_ptr<ScoreModFunc> score_mod);
```

Comment:
Does this need to be a shared_ptr? To me it looks like a non-owning raw pointer would be enough here.